R/cacheDirectories.R

```diff
@@ -19,7 +19,7 @@ setCacheDir = function(cacheDir) {
 #' Set shared cache directory
 #'
 #' Sets global variable specifying the default cache directory for
-#' simpleCacheShared() calls; this function is simply a helper alias for caching
+#' \code{\link{simpleCacheShared}} calls; this function is simply a helper alias for caching
```

Comment:
should this have an `=` then since it's not a package but a function?

Reply:
(@vpnagraj)

Reply:
no i dont think so because it's linking directly to the function name ... the other example was linking to setCacheDir() not setCacheDir